salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Setup MetalK8s sparse loop device %I
+RequiresMountsFor=/var/lib/metalk8s/storage/sparse
```

Comment:
We could add an `AssertFileNotEmpty=/var/lib/metalk8s/storage/sparse/%i` here as well, see https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Conditions%20and%20Asserts (I've been reading too many `systemd` man-pages recently...)

Reply:
Added.